pkgs/build-support/build-pecl.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, lib, php, autoreconfHook, fetchurl, re2c, nix-update-script }:
+{ stdenv, lib, php, autoreconfHook, fetchurl, re2c, nix-update-script }@imports:
```

```suggestion
{ stdenv, lib, php, autoreconfHook, fetchurl, re2c, nix-update-script }@args:
```
that is the standard variable we use for this. Also unless we are using ... @args does not much.

---

The `build-pecl.nix` is a binary lambda function. There's a first set of parameters using `@imports`, and the second one using `@args` already.

Should I change something there?